### DISCUSSION TITLE
How can I fix height of the popover for Picker & Combobox

### DISCUSSION THREAD

**Discussion by NarendraKuruva** [adobe/react-spectrum#6605] [Q&A] [answered]
**Setting height of the Spectrum Combobox dropdown #6500** 

@reidbarber 

Referencing the above discussion
Just in case if I want to fix the height at the worst, what might the the best way to do it?? :thinking: 

Can we handle it in the way below??

Pass another prop `menuHeight` same as `menuWidth`.

Pass it as the height to the popover

Code with the changes: 

```tsx
//combobox.tsx
// .... prev code

let {
      menuTrigger = 'input',
      shouldFlip = true,
      direction = 'bottom',
      align = 'start',
      isQuiet,
      loadingState,
      onLoadMore,
      allowsCustomValue,
      menuWidth: customMenuWidth,
      menuHeight: customMenuHeight,
      name,
      formValue = 'text'
   } = props
//.... remaining

  <Popover
            state={state}
            UNSAFE_style={{...style, height:customMenuHeight}}
//.... rest of the code
/>

```

If not, can suggest a better/best way to handle it

**Comment by reidbarber** [comment, marked as answer]
I take it you are forking, copying, or patch-packaging for your use case. If so, that looks correct, but you may want to use `maxHeight` instead of `height`.

**Reply by NarendraKuruva** [reply]
Got it. Thank you @reidbarber
